std_msgs/Header header
int16[] proximity # length: 24
int16[] force # length: 24
sensor_msgs/Imu imu # Acc(x,y,z) + Gyro(x,y,z)
